src/features/todos.ts

```diff
@@ -1,9 +1,29 @@
+/* eslint-disable import/no-cycle */
```

Comment:
Avoid using 'eslint-disable' for 'import/no-cycle'. Instead, consider refactoring the code to eliminate circular dependencies.